synology_dsm/synology_dsm.py

```diff
@@ -232,14 +234,23 @@ def _request(
 
         return response
 
-    def _execute_request(self, method, url, **kwargs):
+    def _execute_request(self, method, url, params, **kwargs):
         """Function to execute and handle a request."""
         # Execute Request
         try:
             if method == "GET":
-                resp = self._session.get(url, **kwargs)
+                if sys.version_info[0] < 3:
+                    quote = urllib.quote  # pylint: disable=no-member
+                    param_items = params.iteritems()
+                else:
+                    quote = urllib.parse.quote  # pylint: disable=no-member
+                    param_items = params.items()
+                encoded_params = "&".join(
+                    ["%s=%s" % (key, quote(str(value))) for key, value in param_items]
+                )

```

Comment:
Before #33 we uses `urlencode`, I would prefer using that

Reply:
I was using `quote` because `quote_via` is only available in Python 3.5+. Seems `urlencode` from the `future` package is much better.

Reply:
`quote_via` seems still not available in Python 2 since the CI has failed.

Reply:
Ok It's just lint.